src/Certificate/Symmetry/wedderburn.jl

```diff
@@ -10,13 +25,13 @@ function SymbolicWedderburn.decompose(
     return indcs, coeffs
 end
 
-function SymbolicWedderburn.ExtensionHomomorphism(
-    action::SymbolicWedderburn.Action,
-    basis::MB.SubBasis{MB.Monomial},
-)
-    monos = collect(basis.monomials)
-    return SymbolicWedderburn.ExtensionHomomorphism(Int, action, monos)
-end
+#function SymbolicWedderburn.ExtensionHomomorphism(
+#    action::SymbolicWedderburn.Action,
+#    basis::MB.SubBasis{MB.Monomial},
+#)
+#    monos = collect(basis.monomials)
+#    return SymbolicWedderburn.ExtensionHomomorphism(Int, action, monos)
+#end
```

Comment:
yes, the idea of SW is that nobody needs to know about ExtensionHomomorphisms; If you need to interact directly with them, file a bug!

Reply:
Apparently, I have commented it out a few months ago in this PR. I don't remember the details but it seems we are moving in the right direction then ^^